ckanext/nextgeoss/helpers.py

```diff
@@ -489,8 +489,12 @@ def generate_opensearch_query(params):
 
 
 def get_featured_groups_list():
+    '''
+        Returns a list of groups that are setup as featured_groups
+        in the configuration file.
+    '''
     parent_groups = []
-    group_list = config.get('ckan.featured_groups')
+    group_list = config.get('ckan.featured_groups').split()
```

Comment:
Good catch!

The `group_list` was indeed just a string. But from what I understand we want to treat the case when `ckan.featured_groups` is not set at all in the config.

In that case this would fail because the value of `ckan.featured_groups` would be None and we're calling  `.split()` on that:

![image](https://user-images.githubusercontent.com/8862002/59839088-6143a280-9350-11e9-9b22-f8202cc2483e.png)


We can set the default value as an empty string:

```py
group_list = config.get('ckan.featured_groups', '').split()
```

And then if people don't declare any feature groups it will just look empty which makes sense now that I see it:

![image](https://user-images.githubusercontent.com/8862002/59839439-ea5ad980-9350-11e9-8b9c-9f4716e2be76.png)

Reply:
If we wanted to take this further we could replace "View all topics" with "There are no topics yet" or "Add a topic" for these case when the topics are missing. But I think that is worth it's own card.

Reply:
open new ticket in the backlog for this